cmd/destroy/remote.go

```diff
@@ -228,21 +228,8 @@ func (rd *remoteDestroyCommand) createDockerfile(tempDir string, opts *Options)
 
 }
 
-func (rd *remoteDestroyCommand) createDockerignore(cwd, tmpDir string) error {
-	dockerignoreContent := []byte(``)
-	dockerignoreFilePath := filepath.Join(cwd, oktetoDockerignoreName)
-	if _, err := rd.fs.Stat(dockerignoreFilePath); err != nil {
-		if !errors.Is(err, os.ErrNotExist) {
-			return err
-		}
-
-	} else {
-		dockerignoreContent, err = afero.ReadFile(rd.fs, dockerignoreFilePath)
-		if err != nil {
-			return err
-		}
-	}
-	return afero.WriteFile(rd.fs, fmt.Sprintf("%s/%s", tmpDir, ".dockerignore"), dockerignoreContent, 0600)
+func (rd *remoteDestroyCommand) createDockerignore(cwd, tmpDir, manifestPathFlag string) error {
```

Comment:
do we really need this func? Can we call directly `remote.CreateDockerignoreFileWithFilesystem(cwd, tmpDir, manifestPathFlag, rd.fs)`?